map.json

```diff
@@ -33,5 +33,10 @@
 		"id" : "win-bcdedit-ignoreAllFailure",
 		"path" : "src/windows/win-ignoreAllFailures.ps1",
 		"description" : "Ignore errors if there is a failed boot, failed shutdown, or failed checkpoint. The computer will attempt to boot normally after an error occurs."
+	},
+	{
+		"id" : "alar-fki",
```

Comment:
the run-ID should be in the format of:
<os>-<details>
ex) linux-fstab-initrd-kernel or linux-alar-fki

what does alar-fki mean for this case?

Reply:
Hi Sungwoo,

alar: Azure Linux Auto Recover
fki; fstab, kernel, initrd

What don't you like about this ID? If the description is not enough I can make it more descriptive?
I can simply rename it to 'linux-alar-fki' of course to make it visible that this is for Linux only.
The string 'alar' I would like to keep, to make any further developments  part of the alar-construction-kit